keyboards/keebzdotnet/wazowski/keymaps/default/config.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright 2020 keebzdotnet
```

Comment:
This file can be deleted.

Reply:
I have pushed a commit to remove this file